MyLibrary.Domain/Interfaces/IBookRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using MyLibrary.Domain.Entities;
+
+namespace MyLibrary.Domain
+{
+    public interface IBookRepository
+    {
+        object GetAll();
```

Comment:
Not very safe to return objects but i guest it was done when your wrote your test and then made it compile,
To ensure that you dont have to go back and refactor this type of situation when you write your test you can try something like this :
IList<Book> books = _bookRepository.GetAll(parameterName: parameterValue);
this will result in:
IList<Book> GetAll(string parameterName); 
in your interface